docs/etcd/demo_cluster_setup.md

```diff
@@ -1,14 +1,42 @@
 # Set up a demo etcd installation
 
-This guide will help you set up an example etcd cluster that you can use with CN-WAN Operator. For the scope of this demo and keep things simple, the etcd cluster will consist of only one node - or instance.
+**IMPORTANT NOTE**: this guide will only help you create a **demo** cluster so that you can quickly have a working example to use with the CN-WAN Operator and is not intended to be used in production. We strongly encourage you to follow more thorough guides if you want to use etcd in production.
 
-In the first part, *Create the etcd cluster*, we will install and start etcd. The second part, *Make it more secure*, is not mandatory but **highly** suggested.
+The first section focuses on installing etcd in the same Kubernetes cluster as the operator. For a standalone installation jump to *Create the etcd cluster*. The section *Make it more secure*, is not mandatory but **highly** suggested.
 
```

Comment:
Now that there are two ways of installing etcd on this page, I think it would be better to have two links here to redirect people based on where they want to install it.
For example:
- if you want k8s, follow next section
- if you want binary/docker follow this-other-section

Reply:
Sounds good, I rewrote the second paragraph, changed the title for the second section, and added links